docs/source/contributing.rst

```diff
@@ -98,31 +98,31 @@ Consider the following general coding paradigms when contributing:
    silly as the core Python developers are sometimes, there's much to
    be gained by following the basic guidelines listed in PEP 20.
    Without repeating them here, focus on making your additions
-   intuitive, novel, and helpful for DPF-Core and its users.
+   intuitive, novel, and helpful for PyDPF-Core and its users.
 
-   When in doubt, ``import this``
+   When in doubt, ``import this``.
```

Comment:
I added the ending punctuation here but wasn't clear on what the sentence meant.